.github/workflows/examples-telink.yaml

```diff
@@ -63,6 +63,15 @@ jobs:
                     out/telink-tlsr9518adk80d-light/zephyr/zephyr.elf \
                     /tmp/bloat_reports/
 
+            - name: Build example Telink Light Switch App
+              run: |
+                  ./scripts/run_in_build_env.sh \
+                    "./scripts/build/build_examples.py --no-log-timestamps --target-glob 'telink-*' build"
```

Comment:
This builds the same as above: it compiles telink* glob, so it will compile everysingle example.

I guess you should just add the gh_sizes call in a separate run

Reply:
@andy31415 Thank you for revue. Fixed.